PowerPointLabs/PowerPointLabs/PPMouseLegacy.cs

```diff
@@ -1,309 +0,0 @@
-﻿using System;
-using System.Diagnostics;
-using System.Runtime.InteropServices;
-using System.Windows.Forms;
-using System.Drawing;
-using PowerPointLabs;
-using PowerPoint = Microsoft.Office.Interop.PowerPoint;
-using System.Text;
-
-namespace PPExtraEventHelper
-{
-    internal class PPMouseLegacy
```

Comment:
prev all the classes within this file are legacy/deprecated and to be refactored & merged with PPMouse class